com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -113,6 +113,18 @@ public enum LightCookieFormat
         ColorHDR,
     }
 
+    /// <summary>
+    /// Color Buffer format in HDR (only).
+    /// </summary>
+    public enum HDRColorBufferPrecision
+    {
+        /// <summary>Typically R11G11B10f for faster rendering. Recommend for mobile.
+        /// R11G11B10f can cause a subtle blue/yellow banding in some rare cases due to lower precision of blue component.</summary>
+        _32Bits,
```

Comment:
Maybe you can define an [InpsectorName] and a [Tooltip] attributes to nicify the inspector.

Reply:
Added tooltips for the enum options.